docs/book/src/introduction.md

```diff
@@ -16,6 +16,18 @@ hybrid deployments of Kubernetes.  It is built atop the lessons learned from
 previous cluster managers such as [kops](https://github.com/kubernetes/kops) and
 [kubicorn](http://kubicorn.io/).
 
+<aside class="note">
+
+<h1>Cluster API Provider IBM Cloud documentation versions</h1>
+
+This book documents Cluster API Provider IBM Cloud v0.5. For other versions please see the corresponding documentation:
```

Comment:
are you sure the [cluster-api-ibmcloud.sigs.k8s.io](https://cluster-api-ibmcloud.sigs.k8s.io/) points to the v0.5 release?

Reply:
my understanding is that it points to the main branch which is v0.6 version!

Reply:
PR raised - https://github.com/kubernetes/k8s.io/pull/5545 to point default url to latest stable version of doc (v0.5)